CHANGES.md

```diff
@@ -1,9 +1,12 @@
 Version History
 ===============
+## 0.14.12 (06/14/2018)
```

Comment:
will noble.JS ever reach a 1.0 stage?

Reply:
when we start writing the new api service this one will get an honorary version 1.0 as the last version

Reply:
sounds daunting. def worthy of a major version upgrade